troposphere/static/js/components/images/detail/ViewImageDetails.react.js

```diff
@@ -23,32 +22,35 @@ export default React.createClass({
         var profile = stores.ProfileStore.get(),
             image = this.props.image;
 
-        if (profile.id && profile.get("username") === image.get("created_by").username || profile.get("is_staff")) {
+        if (profile.id
+            && profile.get("username")
+            === image.get("created_by").username
+            || profile.get("is_staff")) {
```

Comment:
Indenting by operator is visually nice but doesn't separate by what you want (the tests). 
```
let isStaff = profile.get("is_staff")
let profileUser = profile.get("username")
let imageUser = image.get("created_by").username
if (isStaff || imageUser == profileUser) {
...
}
```
This is much cleaner imo.